components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -3954,6 +3966,16 @@ private Response handleAuthFlowThroughFramework(OAuthMessage oAuthMessage, Strin
                         return Response.status(HttpServletResponse.SC_OK).entity(responseWrapper.getContent()).build();
                     }
                 } else {
+                    try {
+                        String serviceProviderId =
+                                getServiceProvider(oAuthMessage.getRequest().getParameter("client_id"))
+                                .getApplicationResourceId();
+                        requestWrapper.setParameter(SERVICE_PROVIDER_ID, serviceProviderId);
+                    } catch (Exception e) {
+                        // The value is set to be used for branding purposes. Therefore, if an error occurs,
+                        // the process should continue without breaking.
+                        log.debug("Error occurred while getting service provider id.");
```

Comment:
Is this debug or error log?

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2588/commits/84fc98ab729c551156c0593622cdcd3f7cd52360